### ISSUE TITLE
no such column

### ISSUE THREAD

**Issue by TSmiley12** [horilla-opensource/horilla#284]
I just tried installing the system and after running the server i got this error : <html>
<body>

Exception Value: | no such column: base_employeeshiftschedule.is_auto_punch_out_enabled
-- | --
![Screenshot 2024-08-17 155651](https://github.com/user-attachments/assets/10e3a1c4-8d69-4554-a21e-0788a0c2f708)

</body>
</html>

**Comment by horilla-opensource** [comment]
Hi @TSmiley12 ,
Can you run the followings command in the order given below and try again?
```
1.  python3 manage.py migrate base
2. python3 manage.py migrate
```

With Regards,

**Comment by MrcellSbst** [comment]
> Hi @TSmiley12 , Can you run the followings command in the order given below and try again?
> 
> ```
> 1. python3 manage.py migrate base
> 2. python3 manage.py migrate
> ```
> 
> With Regards, **Team Horilla**

hi, i have the same problem, i already did the command, but still got the same problem.

**Comment by horilla-opensource** [comment]
Hi @MrcellSbst ,
If you are starting with a fresh database, please remove the contents from the `__init__.py `file in the base app migrations folder.
ie remove all the contents from the file `base/migrations/__init__.py `and run the above commands.

With Regards,
**Team Horilla**

**Comment by TSmiley12** [comment]
Hello there.
Thank you very much for your support

**Comment by MrcellSbst** [comment]
> Hi @MrcellSbst , If you are starting with a fresh database, please remove the contents from the `__init__.py `file in the base app migrations folder. ie remove all the contents from the file `base/migrations/__init__.py `and run the above commands.
> 
> With Regards, **Team Horilla**

the solution worked, but can't you make this automatically? when I shutdown the server (physically) and when i try to run the server again, this problem pops up

**Comment by horilla-opensource** [comment]
You'll have to run that command only once and it'll be reflected in the database. You don't have to run the command each time you restart the server.
The above commands makes the necessary changes in your database which doesn't affect whether your server is currently running or stopped. So you'll have to execute the command only once.
The code inside the `base/migrations/__init__.py` was meant for existing users because of a major change we made in the inter app dependencies. For new users, this shouldn't have been an issue.

**Comment by horilla-opensource** [comment]
Hi @TSmiley12 ,
We are closing this issue due to inactivity. It seems that there hasn't been any recent activity or discussion, and we believe it may have been resolved or is no longer relevant. 
If you feel this issue is still important and should be addressed, please feel free to reopen it or create a new issue with updated details. 

With Regards,
**Team Horilla**
